src/components/ContextualMenu/ContextualMenu.test.js

```diff
@@ -0,0 +1,18 @@
+import { shallow } from "enzyme";
+import React from "react";
+
+import ContextualMenu from "./ContextualMenu";
+
+describe("Contextual Menu ", () => {
+  it("renders instance but not visible", () => {
+    const wrapper = shallow(<ContextualMenu>Test content</ContextualMenu>);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Snapshots in this case aren't the best idea, you should have an assertion for what you're actually testing.

Reply:
Agree with @hatched, snapshot tests in isolation are not sufficient, but okay to have _both_ I think.